Create Dataset “urban-heating.data”

danielfdsilva opened the pull request:
Automatically generated by Netlify CMS

slesaad commented:
# TODO
Dependency: LST diff dataset publication
- Add a comparison map between LST diff vs SVI index 2018 at the very top of the discovery

slesaad commented on datasets/urban-heating.data.mdx:
```diff
@@ -0,0 +1,238 @@
+      bidx: 1
+      colormap_name: rdbu_r
+      rescale:
+      - -70
+      - 200
```
```suggestion
      - 215
      - 215
```

danielfdsilva commented:
Couple of changes:
- cleaned the commit history by rebasing and squashing (fixed veda ui conflict)
- updated image names to be lowercase
- resized images as they weighed to much
